test/integration/targets/setup_win_psget/tasks/main.yml

```
@@ -10,7 +10,7 @@
   block:
   - name: install PackageManagement
     win_package:
-      path: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
+      path: https://ansible-ci-files.s3.amazonaws.com//test/integration/targets/setup_win_psget/PackageManagement_x64.msi
```

Replace `//` with `/` here and for all following changes:

```suggestion
      path: https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
```

---

fixed thanks